f5/bigip/tm/security/ip_intelligence.py

```diff
@@ -62,8 +62,6 @@ def __init__(self, feed_list_s):
         super(Feed_list, self).__init__(feed_list_s)
         self._meta_data['required_json_kind'] = \
             'tm:security:ip-intelligence:feed-list:feed-liststate'
-        self._meta_data['required_creation_parameters'].update(('partition',))
```

Comment:
is the partition parameter mandatory for this if so only name should have been removed and partition should be added not removed